lib/puppet/util/openssl_loader.rb

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
nit could you move this to `lib/puppet/ssl/openssl_loader.rb`?

Reply:
absolutely